--- a/pico/composer.c
+++ b/pico/composer.c
@@ -794,7 +794,7 @@
 		    len += lmp->fname ? strlen(lmp->fname) : 0;
 
 		    if(len+3 > sizeof(buf)){
-			bfp = malloc(len+3);
+			/*bfp = malloc(len+3);*/
 			space = len+3;
 			if((bfp=malloc(len+3)) == NULL){
 			    emlwrite("\007Can't malloc space for filename",
@@ -1279,7 +1279,7 @@
 			len += strlen(lmp->size);
 
 			if(len+3 > sizeof(buf)){
-			    bfp = malloc(len+3);
+			    /*bfp = malloc(len+3);*/
 			    space = len+3;
 			    if((bfp=malloc(len+3)) == NULL){
 				emlwrite("\007Can't malloc space for filename",
