Index: alpine-2.02/imap/src/osdep/unix/flocklnx.c
===================================================================
--- alpine-2.02.orig/imap/src/osdep/unix/flocklnx.c	2010-10-02 04:37:57.000000000 -0400
+++ alpine-2.02/imap/src/osdep/unix/flocklnx.c	2011-11-11 08:58:18.297338885 -0500
@@ -57,7 +57,7 @@
     case ENOLCK:		/* lock table is full */
       sprintf (tmp,"File locking failure: %s",strerror (errno));
       mm_log (tmp,WARN);	/* give the user a warning of what happened */
-      if (!logged++) syslog (LOG_ERR,tmp);
+      if (!logged++) syslog (LOG_ERR, "%s", tmp);
 				/* return failure if non-blocking lock */
       if (op & LOCK_NB) return -1;
       sleep (5);		/* slow down in case it loops */
Index: alpine-2.02/alpine/mailcmd.c
===================================================================
--- alpine-2.02.orig/alpine/mailcmd.c	2010-10-02 04:37:57.000000000 -0400
+++ alpine-2.02/alpine/mailcmd.c	2011-11-11 09:02:29.357346728 -0500
@@ -7879,7 +7879,7 @@
 
 	      case 13 :			/* Match my addresses */
 		me++;
-		snprintf(sstring, sizeof(sstring), not ? _(dont_match_me) : _(match_me));
+		snprintf(sstring, sizeof(sstring), "%s", not ? _(dont_match_me) : _(match_me));
 		continue;
 
 	      case 14 :			/* Subject: default */
Index: alpine-2.02/pith/ldap.c
===================================================================
--- alpine-2.02.orig/pith/ldap.c	2010-10-02 04:37:57.000000000 -0400
+++ alpine-2.02/pith/ldap.c	2011-11-11 09:02:03.401345919 -0500
@@ -777,7 +777,7 @@
 	 */
 	switch(args){
 	  case 0:
-	    snprintf(filter, sizeof(filter), filt_format);
+	    snprintf(filter, sizeof(filter), "%s", filt_format);
 	    break;
 	  case 1:
 	    snprintf(filter, sizeof(filter), filt_format, scp);
